### PR TITLE
Added Polish translation

### PR DESCRIPTION
**Pull request by PawelCode**

### CODE REVIEW

**Comment by Mistrick** [comment]
Can anyone check the translation?

**Comment by Happy2906** [comment]
As far as I can see, the translation is correct.

File: cstrike/addons/amxmodx/data/lang/mapmanager.txt [hunk 1]
```diff
@@ -45,6 +45,53 @@ MAPM_SECOND_VOTE = Second vote.
...
+MAPM_MENU_NEXT = Dalej
+MAPM_MENU_EXIT = Wyjdz
+MAPM_HUD_TIMER = Glosowanie rozpocznie sie za %d
+MAPM_MENU_VOTE_RESULTS = Rezultat glosowania
+MAPM_MENU_CHOOSE_MAP = Wybierz mape
+MAPM_MENU_EXTEND = Przedluz
+MAPM_MENU_LEFT = Koniec glosowania za:
+MAPM_CHOSE_EXTEND = %s^1 Wybral^4 przedluzenie^1 aktualnej mapy.
+MAPM_CHOSE_MAP = %s^1 wybral^4 %s^1.
+MAPM_NOBODY_VOTE = Nikt nie oddal glosu. .
```
Comment (Mistrick) [inline cstrike/addons/amxmodx/data/lang/mapmanager.txt:80]: remove second dot

**Review by Mistrick — CHANGES_REQUESTED** [review]

File: cstrike/addons/amxmodx/data/lang/mapmanager.txt (hunk 1 above)
Comment (PawelCode) [reply cstrike/addons/amxmodx/data/lang/mapmanager.txt]: Done

**Pull request merged**